packages/flutter/lib/src/material/slider.dart

```diff
@@ -170,12 +170,29 @@ class Slider extends StatefulWidget {
 }
 
 class _SliderState extends State<Slider> with TickerProviderStateMixin {
+  _SliderState() {
+    _reactionController = new AnimationController(
+      duration: kRadialReactionDuration,
+      vsync: this,
+    );
+  }
+
   void _handleChanged(double value) {
     assert(widget.onChanged != null);
     widget.onChanged(value * (widget.max - widget.min) + widget.min);
   }
 
   @override
+  void dispose() {
+    _reactionController?.dispose();
```

Comment:
why `?`? Since you set it when the State is created, and dispose of it when it's finished, there's no way it can be anything but non-null, no?